components/General/validation.ts

```diff
@@ -16,7 +14,7 @@ export const removeEmojis = (passwordInput: string) => {
  * @param password
  * @returns true if password contains at least one letter, false otherwise
  */
-export const validatePasswordContainsLetter = (password: string) => {
+export const validatePasswordContainsLetter = (password: string): boolean => {
```

Comment:
`regex.text` returns a boolean and such the return type of `validatePasswordContainsLetter` can easily be inferred. No need to add redundant types.

Also valid for the functions below.